coderd/database/dbauthz/dbauthz.go

```diff
@@ -587,8 +587,8 @@ func (q *querier) SoftDeleteTemplateByID(ctx context.Context, id uuid.UUID) erro
 }
 
 func (q *querier) GetUsersWithCount(ctx context.Context, arg database.GetUsersParams) ([]database.User, int64, error) {
```

Comment:
Do we not need this function anymore? Can it just use `GetUsers` instead?

Reply:
Yup! Good catch, it was an extra function. I should write a unit test to error on extra funcs :thinking:

Reply:
Ah I see, I did this thinking dbauthz wouldn't have to match the db.Store interface and just made the function signature nicer. I'll still remove it though